src/drawing.rs

```diff
@@ -627,14 +627,40 @@ impl Drawing {
     pub fn save_svg<P: AsRef<Path>>(&self, path: P) -> Result<(), ExportError> {
         block_on(self.drawing.save_svg(path))
     }
+
+    /// Destroys underlying window and drops self.
+    ///
+    /// Subsequent commands to turtle, created using [`Drawing::add_turtle`], might panic.
+    ///
+    /// ```rust
+    /// use turtle::Drawing;
+    ///
+    /// let mut drawing = Drawing::new();
+    /// let mut turtle = drawing.add_turtle();
+    ///
+    /// turtle.set_speed(2);
+    /// turtle.right(90.0);
+    /// turtle.forward(50.0);
+    ///
+    /// // close window
+    /// drawing.destroy();
+    ///
+    /// // this will panic!
+    /// // turtle.forward(100.0)
+    /// ```
+    pub fn destroy(self) {
```

Comment:
Could you mark this method as unstable? I want to make sure we don't stabilize this until we've had a chance to discuss the name of the method in #181.

Example of how to do that: https://github.com/sunjay/turtle/blob/bb20f85684ca2ababc9763fe081064a157448626/src/turtle.rs#L259-L260